plugins/inputs/gnmi/gnmi.go

```diff
@@ -47,6 +47,7 @@ type GNMI struct {
 	Prefix           string            `toml:"prefix"`
 	Target           string            `toml:"target"`
 	UpdatesOnly      bool              `toml:"updates_only"`
+	VendorSpecific   []string          `toml:"vendor_specific"`
```

Comment:
Can we please check the users choice against the list of supported extensions!? Otherwise the user might specify some unsupported extension but it's not handled nor generates an error.

Reply:
This was done in Init(). Sample error message:

2023-04-25T14:27:28Z E! [telegraf] Error running agent: could not initialize input inputs.gnmi: Unsupported vendor_specific option': unknown choice foo